README.md

```diff
@@ -105,7 +105,7 @@ fetch('<issuer>/.well-known/trust-token', {
 ```
 
 
-If there are no tokens available for the given issuer, the returned promise rejects with an error. Otherwise, it invokes the PrivacyPass redemption protocol against the issuer, with the token (potentially, if specified by an extension, along with associated redemption metadata) attached in the Sec-Trust-Token request header. The issuer can either consume the token and act based on the result, optionally including a Redemption Record (RR) in the Sec-Trust-Token response header to provide a redemption attestation to forward to other parties. Additionally, the issuer may include the `Sec-Trust-Token-Lifetime` header in the response to indicate to the UA how long (in seconds) the redemption record should be cached for. If the header is omitted, the expiry of the Trust Token key will be used instead to decide when to discard the redemption record.
+If there are no tokens available for the given issuer, the returned promise rejects with an error. Otherwise, it invokes the PrivacyPass redemption protocol against the issuer, with the token (potentially, if specified by an extension, along with associated redemption metadata) attached in the Sec-Trust-Token request header. The issuer can either consume the token and act based on the result, optionally including a Redemption Record (RR) in the Sec-Trust-Token response header to provide a redemption attestation to forward to other parties. Additionally, the issuer may include the `Sec-Trust-Token-Lifetime` header in the response to indicate to the UA how long (in seconds) the redemption record should be cached for. If the header is omitted, the lifetime of the redemption record will be tied to the lifetime of the Trust Token key.
```

Comment:
This behavior is definitely a little subtle, so I'm not sure there _is_ a way to explain it in a specific and concise way... that said:

> lifetime of the redemption record will be tied to the lifetime of the Trust Token key

Which key? Perhaps "the Trust Token verification key associated with the redeemed token," or, if we want to be really specific, "the Trust Token verification key that confirmed the redeemed token's issuance"?